### PR TITLE
Modify Bartmodel Docstring

### PR DESCRIPTION
**Pull request by huhuiwen99**

### CODE REVIEW

**Review by yingyibiao — COMMENTED** [review]
image
class BartEncoder的forward方法api文档缺失

**Review by yingyibiao — DISMISSED** [review]

File: paddlenlp/transformers/bart/modeling.py [hunk 1]
```diff
@@ -181,6 +188,21 @@ def __init__(self,
...
+                See :class:`BartModel`.
+            attention_mask (Tensor, optional):
+                See :class:`BartModel`.
+
+        Returns:
+            Tensor: Returns tensor `encoder_output`, which is the output at the last layer of the model.
+            Its data type should be float32 and has a shape of [batch_size, sequence_length, hidden_size].
+
+        """
     def forward(self, input_ids=None, attention_mask=None, **kwargs):
```
Comment (yingyibiao) [inline paddlenlp/transformers/bart/modeling.py]: def forward(self, input_ids=None, attention_mask=None, **kwargs):
这一行重复了两次

Comment (huhuiwen99) [reply paddlenlp/transformers/bart/modeling.py]: done

File: paddlenlp/transformers/bart/modeling.py [hunk 2]
```diff
@@ -375,6 +547,11 @@ def __init__(self,
         self.out_proj = nn.Linear(inner_dim, num_classes)
 
     def forward(self, hidden_states):
+        """
+        Args:
+            hidden_states (Tensor):
+                Hidden states of the classification model.
+        """
```
Comment (yingyibiao) [inline paddlenlp/transformers/bart/modeling.py:554]: 缺少返回值

**Review by yingyibiao — APPROVED** [review]

**Pull request merged**